src/components/SideBar/SideBar.tsx

```diff
@@ -15,6 +14,7 @@ export interface SideBarProps {
   onCloseClick: () => void;
   isSideBarOpen: boolean;
   SideBarLogo: JSX.Element;
+  styles: Record<string, string>
```

Comment:
Allows us to pipe in styles from a parent component so that this component and its sibling (Navbar) can interact